CONTRIBUTING.md

```diff
@@ -1,165 +1,77 @@
 # Contributing
 
-We welcome feedback and help with Psammead. By participating in Psammead, you agree to abide by the [Code of Conduct](https://github.com/bbc/psammead/blob/latest/CODE_OF_CONDUCT.md). Please take a moment to read it.
-
-We appreciate all forms of contribution - not just code. Contribution can include adding documentation, requesting clarification, making typo corrections and much more.
-
-Psammead has a "Definition of Done" which applies to changes proposed to its main branch, `latest`. When you make a pull request, please check that your contribution matches this.
-
-## Definition of Done
-
-**Component Structure**
-
-As discussed in the [README](https://github.com/bbc/psammead/blob/latest/README.md), Psammead is a library of shared components following the [container / components pattern](https://medium.com/@dan_abramov/smart-and-dumb-components-7ca2f9a7c7d0). Whether you are creating a component from scratch, or preparing to move an existing React component to Psammead, make sure that your component is presentational and reusable. The component should accept props if needed and implement minimal logic. A presentational component shouldn’t handle either fetching or mutation of data.
-
-A component should have this file structure:
-
-```
-├── README.md
-├── CHANGELOG.md
-├── package.json
-├── package-lock.json
-├── src
-│   ├── __snapshots__
-│   │   └── index.test.jsx.snap
-│   ├── index.jsx
-│   ├── index.stories.jsx
-│   └── index.test.jsx
-```
+## Documentation index
+Please familiarise yourself with our:
+- [Code of conduct](https://github.com/bbc/psammead/blob/latest/CODE_OF_CONDUCT.md)
+- [Code Standards and Ways of Working](https://github.com/bbc/simorgh/blob/latest/Code-Standards-and-Ways-of-Working.md)
+- [Contributing guidelines](https://github.com/bbc/psammead/blob/latest/CONTRIBUTING.md) (you are here)
+- [Github Project Board Guide](https://github.com/bbc/simorgh/blob/latest/docs/Project-Board-Guide.md)
+- [Primary README](https://github.com/bbc/psammead/blob/latest/README.md)
+- [Talos (package bumping bot)](https://github.com/bbc/psammead/blob/latest/scripts/talos/README.md)
+- [Use/consumption of Psammead packages guidelines and package list](https://github.com/bbc/psammead/blob/latest/packages/README.md)
 
-The `src/` folder is necessary for separation of concerns, as a `dist/` folder containing the transpiled version of the component is generated at publish time. The `dist/` folder will contain the transpiled version of any `.jsx` files in the `src/` folder (saved as ES5 `.js`), with the exception of `*.stories.jsx` and `*.test.jsx` files.
+NB there is further documentation colocated with relevant packages and code. The above list is an index of the top-level documentation of our repo (and our sibling repo [Simorgh](https://github.com/bbc/simorgh)).
 
-NB: the `__snapshots__` directory is optional.
+## Overview
```

Comment:
Does this section need to mention what `Utilities` are used for?

Reply:
@HarveyPeachey I'm open to that. But beyond what the name implies and without listing them (which was done and proved it will always be out of date as several weren't mentioned at all) what do you suggest?

Reply:
@benjaminhobbs Maybe at the very least just linking to the readme for utilities?

Reply:
Talked elsewhere, that info is partly cut and partly moved to packages readme, but as @HarveyPeachey is about to create a new utility new will add docs in another PR if he feels it's not adequately described here.